<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(100, 100, 100);
font: 75 italic 17pt &quot;PibotoLt&quot;;
border:5px solid rgb(100,100,100);</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="time">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);
font:18pt &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="id">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 30pt &quot;Quicksand&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>50</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="true_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 255, 0);
font: 18pt &quot;Quicksand&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="boder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border:10px solid rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yes">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);
font: 75 italic 30pt &quot;PibotoLt&quot;;
border:5px solid rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string>YES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 75 italic 30pt &quot;PibotoLt&quot;;
border:5px solid rgb(255,0,0);</string>
   </property>
   <property name="text">
    <string>NO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 100, 255);
font: 75 italic 15pt &quot;PibotoLt&quot;;
border:5px solid rgb(0,100,255);</string>
   </property>
   <property name="text">
    <string>FP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="comeBack">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>419</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);
font: 75 italic 20pt &quot;PibotoLt&quot;;
border:5px solid rgb(0,255,0);</string>
   </property>
   <property name="text">
    <string>COME BACK</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
